Civi/Payment/PropertyBag.php

```diff
@@ -591,6 +609,27 @@ public function setBillingCounty($input, $label = 'default') {
     return $this->set('billingCounty', $label, (string) $input);
   }
 
+  /**
+   * BillingStateProvince getter.
+   *
+   * @return string
+   */
+  public function getBillingStateProvince($label = 'default') {
+    return $this->get('billingStateProvince', $label);
+  }
+
+  /**
+   * BillingStateProvince setter.
+   *
+   * Nb. we can't validate this unless we have the country ID too, so we don't.
+   *
+   * @param string $input
+   * @param string $label e.g. 'default'
+   */
+  public function setBillingStateProvince($input, $label = 'default') {
+    return $this->set('billingStateProvince', $label, (string) $input);
+  }
+
```

Comment:
Add in billingStateProvince and getter/setter which is required for full billingAddress coverage.

Reply:
Merged via #21583